### DISCUSSION TITLE
How to completely destroy the entire scene?

### DISCUSSION THREAD

**Discussion by 3261564019** [enable3d/enable3d#373] [Help / Questions] [answered]
How to completely destroy the entire scene? I hope to provide a case study. By the way, what are the configurable items of this config object that I did not find in the documentation
`  const config = { scenes: [MainScene] } 
  PhysicsLoader('/lib', () => new Project(config))`

**Comment by yandeu** [comment, marked as answer]
Take a look at the [Project constructor](https://github.com/enable3d/enable3d/blob/575d73c49a9f16ef8cde6b17508f71d615e5e391/packages/enable3d/src/project.ts#L22) and the Scene3DConfig and ThreeGraphicsConfig interface.

I believe there is no complete destroy function.
